### ISSUE TITLE
[Feature Request]: Stop notification sounds on boards if you are in DND on smartphone 

### ISSUE THREAD

**Issue by Dtrieb** [meshtastic/Meshtastic-Android#1483]
### Tell us your idea.

I'd like the app to stop notification sounds on boards if you are in DND on Smartphone.

P.e. if you set your phone on DND and forgot that the tracker plays it's tune, it very annoying in a meeting.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by todd2982** [comment]
This would involve rebooting the node each time the phone's DND setting was toggled.  This would cause the T1000e to play the boot-up tone each time.  Is this something you would still like somebody to look into?

**Comment by jamesarich** [comment]
I don't think it makes sense to tie these things together, they are both individually settable depending on where/how you'd like to be notified.
